packages/expo-updates/android/src/main/java/expo/modules/updates/UpdatesModule.kt

```diff
@@ -246,6 +252,42 @@ class UpdatesModule(
     }
   }
 
+  @ExpoMethod
+  fun readLogEntriesAsync(maxAge: Long, promise: Promise) {
+    AsyncTask.execute {
+      val reader = UpdatesLogReader()
+      val date = Date()
+      val epoch = Date(date.time - maxAge)
+      val result = reader.getLogEntries(epoch).map {
+        val entry = UpdatesLogEntry.create(it)
+        val entryBundle = Bundle()
+        entryBundle.putLong("timestamp", entry.timestamp)
+        entryBundle.putString("message", entry.message)
+        entryBundle.putString("code", entry.code)
+        entryBundle.putString("level", entry.level)
+        if (entry.updateId != null) {
+          entryBundle.putString("updateId", entry.updateId)
+        }
+        if (entry.assetId != null) {
+          entryBundle.putString("assetId", entry.assetId)
+        }
+        if (entry.stacktrace != null) {
+          entryBundle.putStringArray("stacktrace", entry.stacktrace.toTypedArray())
+        }
+        entryBundle
```

Comment:
can put this in `apply` block

```
Bundle().apply {
   putString(...)
   ...
}
```
(this removes the need for local variables and keeps things more functional)

Reply:
Done